src/styles/index.scss

```diff
@@ -2,2 +1,127 @@
+$product-info-color: #616070;
+$main-color: #060b35;
+$btn-color: #00acdc;
+
+* {
+  box-sizing: border-box;
   margin: 0;
+  padding: 0;
+  font-family: Roboto, sans-serif;
+}
+
+body {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.container {
+  width: 200px;
+  height: 406px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin-bottom: 40px;
+    margin-inline: 19px;
+  }
+
+  &__product-info {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+    margin-bottom: 16px;
+
+    .product-info-title {
+      color: $main-color;
+      font-size: 12px;
+      font-weight: 500;
+      line-height: 18px;
+    }
+
+    .product-info-text {
+      color: $product-info-color;
+      font-size: 10px;
+      line-height: 14px;
+    }
+  }
+
+  &__stars-reviews {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 24px;
+
+    .container__stars-reviews-star {
+      display: flex;
+    }
+
+    .container__stars-reviews-star-icon {
+      width: 16px;
+      height: 16px;
+      background-size: center;
+      background-image: url('../images/star.svg');
+      background-repeat: no-repeat;
+      background-position: center;
+      margin-right: 4px;
+
+      &:nth-child(-n + 4) {
+        background-image: url('../images/star-active.svg');
+      }
+    }
+
+    .review-text {
+      color: $main-color;
+      font-size: 10px;
+      line-height: 14px;
+      display: flex;
+      align-items: flex-end;
+    }
+  }
+
+  &__product-price {
+    width: 100%;
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    .price-text {
+      color: $product-info-color;
+      font-size: 12px;
+      line-height: 18px;
+    }
+
+    .cost {
+      color: $main-color;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__btn {
+    padding: 11px 69px;
```

Comment:
The horizontal padding for the button is quite large. Ensure that this padding aligns with the design requirements. If the button is supposed to adjust to the text size, consider using `padding-left` and `padding-right` for better control.